frontend/app/entry.server.tsx

```diff
@@ -27,11 +27,10 @@ const abortDelay = 5_000;
 const instrumentationService = getInstrumentationService();
 instrumentationService.startInstrumentation();
 
-const log = getLogger('entry.server');
-
 const { ENABLED_MOCKS } = getEnv();
 
 if (ENABLED_MOCKS.length > 0) {
+  const log = getLogger('entry.server');
```

Comment:
This is still a side-effect.

I don't know what the best way to handle this is. I think msw needs to be started/init'd as early as possible here, so there might be no way to avoid this side effect.

@sebastien-comeau any comments?

Reply:
Well in this case we can't do that much because it's on the main thread of the application.